data/examples/other/haddock-sections-out.hs

```diff
@@ -0,0 +1,7 @@
+-- $weird #anchor#
```

Comment:
This is a regression – before this commit it was rendering it as `-- $weird#anchor#`. I'm not sure how it got broken.

Reply:
It looks like named sections and [anchors](https://haskell-haddock.readthedocs.io/en/latest/markup.html#anchors) have nothing in common. Why do you think that it is a regression? Is this example from a real code base where such a combination serves some purpose?